### PR TITLE
[RFC][NI] Add autocomplete component

### PR DESCRIPTION
**Pull request by montoias**
NI

### CODE REVIEW

File: addon/components/uni-autocomplete.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  actions: {
+    onSelected(option) {
+      this._changeValue(option.matchedValues.get('firstObject').capitalize());
+
+      this.get('onSelected')(option);
+    }
+  },
+
+
```
Comment (AndreJoaquim) [inline addon/components/uni-autocomplete.js]: remove empty line

File: app/components/uni-autocomplete.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-uniq/components/uni-autocomplete';
```
Comment (AndreJoaquim) [inline app/components/uni-autocomplete.js]: newline pls

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

**Review by AndreJoaquim — APPROVED** [review]

File: addon/styles/partials/_uni-autocomplete.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
+.uni-autocomplete {
+    position: relative;
```
Comment (mmelo) [inline addon/styles/partials/_uni-autocomplete.scss]: uniQ columns already add position: relative - so please remove this line

File: addon/styles/partials/_uni-autocomplete.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    }
+}
+
+.options {
+    @include column(12);
+    height: 210px;
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    overflow: scroll;
```
Comment (mmelo) [inline addon/styles/partials/_uni-autocomplete.scss]: `overflow-x: hidden;
overflow-y: scroll;`

**Review by mmelo — COMMENTED** [review]
reviewed

**Pull request merged**